scripts/us_census/pep/monthly_population_estimate/test_data/expected_USA_Population_Count.tmcf

```diff
@@ -0,0 +1,44 @@
+Node: E:Population_Count_USA->E0
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:Count_Person_InUSArmedForcesOverseas

```

Comment:
Is there no total population count?

Reply:
Population Counts are done by the categories available like ResidesInHousehold, USResident, Civilian, etc according to the available data